go.mod

```diff
@@ -6,16 +6,16 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/allegro/bigcache v1.2.1 // indirect
 	github.com/aristanetworks/goarista v0.0.0-20201012165903-2cb20defcd66 // indirect
+	github.com/bartekn/go-bip39 v0.0.0-20171116152956-a05967ea095d // indirect
 	github.com/btcsuite/btcd v0.21.0-beta // indirect
 	github.com/cespare/cp v1.1.1 // indirect
-	github.com/cosmos/cosmos-sdk v0.39.1
+	github.com/cosmos/cosmos-sdk v0.40.0-rc0
```

Comment:
Any reason you arent using the latest rc? ie:`https://github.com/cosmos/cosmos-sdk/tree/v0.40.0-rc3`

Reply:
https://github.com/cosmos/cosmos-sdk/tree/v0.40.0-rc3

Reply:
no specific reason. Just because startport generate 0.40.0-rc0. Lack of information, we must rely on at least something. But I think all this PR is just wrong approach. It is better to create separated application (v0.40.0-rc0) from scratch and start from this

Reply:
YEah ok. For now we'll just put this onhold as its a lot bigger than we want to take on right now.